Source/Core/Gen4/Searchers/WildSearcher4.cpp

```diff
@@ -378,7 +378,7 @@ std::vector<WildState> WildSearcher4::searchInitialSeeds(const std::vector<WildS
 std::vector<WildState> WildSearcher4::normalMethodJ(WildState state, u32 seed) const
 {
     std::vector<WildState> states;
-    state.setLead(Lead::None);
+    lead == Lead::CompoundEyes? state.setLead(Lead::CompoundEyes) : state.setLead(Lead::None);
```

Comment:
Probably just be better to do `state.setLead(lead)`

Reply:
done